src/content/api/asset-types.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Asset Types
+description: Asset types supported for payment
+draft: true
+nav:
+  path: API
+  order: 10 
```

Comment:
Should this be 12? How are you counting this?
It's the 12 based on the existing docs site.

Reply:
Updated